scss/widgets/generic/treeView/_index.scss

```diff
@@ -35,7 +35,7 @@ $generic-treeview-toggle-item-visibility-offset: -4px;
       overflow: visible;
     }
 
-    .dx-treeview-node-container:has(.dx-treeview-item-with-custom-expander-icon) {
+    .dx-treeview-rtl-custom-expander-icon-container-offset {
```

Comment:
Let's rename: `.dx-treeview-custom-expander-icon-item-container`.
Do you agree, @ksercs?